IPs/RSKIP152.md

```diff
@@ -0,0 +1,50 @@
+# Title
+
+|RSKIP          |152           |
+| :------------ |:-------------|
+|**Title**      |CHAINID Opcode |
+|**Created**    |19-NOV-19 |
+|**Author**     |SMS |
+|**Purpose**    |Sca, Usa |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Adopted |
+
+## Abstract
+
+The purpose of this RSKIP is to describe the new opcode include in the next rskJ reslease: The CHAINID opcode.
+
+## Motivation
+
+In order to use the chain ID to prevent replay attacks between different chains. It would be a great benefit to have the same possibility inside smart contracts when handling signatures.
```

Comment:
It would be good if you can extract some paragraph of rationale and write it here, so Motivation becomes a little bit more clear

Reply:
Incorrect grammar. Maybe this:
"This RSKIP proposes the addition of the CHAINID opcode to replay attacks between different chains, specially for replay protection of ECDSA-signed messages in smart contracts."